_docs/installation.md

```diff
@@ -0,0 +1,171 @@
+
+First, clone the [simple-games-api](https://github.com/simplworld/simpl-games-api) repository:
+
+```bash
+git clone https://github.com/simplworld/simpl-games-api
+```
+
+The `simpl-games-api` should always be installed first and will run the entire time.
+
+`cd` into the `simpl-games-api` repo and start Docker.
+
+```bash
+cd simpl-games-api
+docker-compose up
+```
+
+Open another terminal window. In that window, use a `make` script to log into the container, which is named `api`.
+
+```bash
+make shell
+```
+
+You will see in the window that you are now logged into the container.
+
+```bash
+Creating simpl-games-api_api_run ... done
+root@b2397c235f68:/code# 
+```
+
+Create a `superuser` and `staff user` by running `./manage.py simpl_setup` 
+
+```bash
+root@b2397c235f68:/code# ./manage.py simpl_setup
+```
+
+`superuser` is used to access the data in the Simpl API's Django admin. `staff user` is used by Simpl model services to connect to the Simpl API.
+
+Create a `superuser` when prompted. `superuser` can have any email address and password.
+
+```bash
+=== Setup Simpl API ===
+
+We found no superusers, please set one up below
+--- Creating a superuser ---
+Please enter email address for superuser: 
+```
+
+Create a `staff user` when prompted. `staff user` must use the email address `simpl@simpl.world` and password `simpl`, which correspond to the email address and password in the model service `settings.py` file.
+
+```bash
+We found no staff users, you will need one of these for your model service to connect to the Simpl API
+This is typically setup as a setting `SIMPL_GAMES_AUTH` from the environment variables `SIMPL_USER` and `SIMPL_PASS`
+
+--- Creating a staff user ---
+Please enter email address for staff user: 
+```
+
+`simpl-games-api` is now up and running. You can now exit and close that window.
+
+```bash
+exit
+```
+
+In your browser, open up `http://localhost:8100/admin/`. The Django admin login page should appear.
+
+You can now login using either the `superuser` or `staff user` credentials.
+
+With the `simpl-games-api` still running in the first window, open another window and `cd` back into your user directory.
+
+```bash
+cd ..
+```
+
+Clone the [simple-calc-model](https://github.com/simplworld/simpl-calc-model) repository and `cd` into it.
+
+```bash
+git clone https://github.com/simplworld/simpl-calc-model
+cd simpl-calc-model
+```
+
+We will now get `simpl-calc-model` up and running. 
+
+```bash
+docker-compose up
+```
+
+`docker-compose up` will need to be run twice. The first time you run it, you will see an error, because the model service will expect a game, but the game doesn't exist yet.
+
+```bash
+#snippet
+genericclient_base.exceptions.ResourceNotFound: No `games` found for {'slug': 'simpl-calc'}
+```
+
+Create a third window. You will now run `simpl-calc-model` and create the game, including the database.
+
```

Comment:
You will run a `simpl-calc-model` Django management command to create the game in the database. The database already exists.

Reply:
I've corrected that.